Mathlib/CategoryTheory/Limits/FullSubcategory.lean

```diff
@@ -39,6 +39,20 @@ section
 
 variable {C : Type u} [Category.{v} C] {J : Type w} [Category.{w'} J] {P : C → Prop}
 
+theorem closedUnderLimitsOfShape_of_limit (hP : ∀ {X Y : C}, (X ≅ Y) → P X → P Y)
```

Comment:
Could you use https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/ClosedUnderIsomorphisms.html#CategoryTheory.ClosedUnderIsomorphisms here?

Reply:
Yes, good idea!